templates/jenkins/files/README.md

```diff
@@ -1,19 +1,37 @@
-# Jenkins template for Platform.sh
+# Jenkins for Platform.sh
 
-This project provides a starter kit for Jenkins projects hosted on [Platform.sh](http://platform.sh).
+This project provides a starter kit for Jenkins projects hosted on Platform.sh.
 
-We include recommended .platform.app.yaml and .platform files that should suffice for most use cases. You are free to tweak them as needed for your particular site.
+Jenkins is an open source automation server written in Java. Jenkins helps to automate the non-human part of the software development process, with continuous integration and facilitating technical aspects of continuous delivery.
 
-## Platform.sh configuration files
+## Services
+
+* Java 8
+
+## Post-install
+
+1. After running a Jenkins instance by Platform.sh, the post-installation setup wizard begins.
+2. When you first access a new Jenkins instance, you are asked to unlock it using an automatically-generated password.
+3. Browse to the Jenkins service link and wait until the Unlock Jenkins page appears.
+4. From the Jenkins console log output, copy the automatically-generated alphanumeric password 
+5. Access the Jenkins instance with the (ssh command)[https://docs.platform.sh/development/access-site.html#accessing-the-application-with-ssh]
+6. Read the information from the file `/app/.jenkins/secrets/initialAdminPassword` with the following command:
+```shell
+cat /app/.jenkins/secrets/initialAdminPassword
+```
+7. On the Unlock Jenkins page, paste this password into the Administrator password field and click Continue.
+
+## Customizations
+
+The following files and additions make the Jenkins work.  If using this project as a reference for your own existing project, replicate the changes below to your project.
 
 * [`.platform/routes.yaml`](.platform/routes.yaml): Platform.sh allows you to define the [routes](https://docs.platform.sh/configuration/routes.html).
 * [`.platform/services.yaml`](.platform/services.yaml):  Platform.sh allows you to completely define and configure the topology and [services you want to use on your project](https://docs.platform.sh/configuration/services.html).
 * [`.platform.app.yaml`](.platform.app.yaml): You control your application and the way it will be built and deployed on Platform.sh [via a single configuration file](https://docs.platform.sh/configuration/app-containers.html).
 
 ## References
 
-
 * [Maven](https://maven.apache.org/)
 * [Jenkins](https://jenkins.io/) 
 * [Platform Documentation](https://docs.platform.sh/)
-* [Java at Platform.sh](https://docs.platform.sh/languages/java.html)
```

Comment:
I don't think the link to the docs root is necessary; just the language link and application home page link, in most cases.

Reply:
Done, thank you